src/utils/DataDefinitionLanguage.js

```diff
@@ -126,6 +126,18 @@ class DataDefinitionLanguage {
     result += ");";
     return result;
   }
+
+  saveRenamedNodeTable(oldName, newName) {
```

Comment:
The two functions below should be merged. Also, it should just be `renameTable`.